### PR TITLE
qemu: bios load OP-TEE image v2

### PR DESCRIPTION
**Pull request by jenswi-linaro**
Bios loads OP-TEE image v2 binaries instead of the v1 format.

Signed-off-by: Jens Wiklander <jens.wiklander@linaro.org>

Depends on https://github.com/linaro-swg/bios_qemu_tz_arm/pull/13

### CODE REVIEW

File: qemu.mk [hunk 1]
```diff
@@ -41,13 +41,15 @@ define bios-qemu-common
...
-		BIOS_SECURE_BLOB=$(OPTEE_OS_BIN) \
 		PLATFORM_FLAVOR=virt
 endef
 
 bios-qemu: update_rootfs optee-os
+	ln -sf $(OPTEE_OS_HEADER_V2_BIN) .
+	ln -sf $(OPTEE_OS_PAGER_V2_BIN) .
+	ln -sf $(OPTEE_OS_PAGEABLE_V2_BIN) .
+	ln -sf $(LINUX_PATH)/arch/arm/boot/zImage .
+	ln -sf $(GEN_ROOTFS_PATH)/filesystem.cpio.gz rootfs.cpio.gz
```
Comment (etienne-lms) [inline qemu.mk]: This fills the **build/** directory with the optee images.
Could we do as the qemu/v8, store bins in a `$(BIOS_QEMU_PATH)/build/release` and run qemu for the later?

Comment (jenswi-linaro) [reply qemu.mk]: Except that we don't have any such path here as we're not building ARM-TF.
How about putting it in `out/` or `out/bin` instead?

Comment (etienne-lms) [reply qemu.mk]: out/ or out/bin are fine. Thanks.

**Comment by jenswi-linaro** [comment]
Update

**Comment by etienne-lms** [comment]
`Reviewed-by: Etienne Carriere <etienne.carriere@linaro.org>` (/!\ dependency)

**Comment by jenswi-linaro** [comment]
Tag applied

**Pull request merged**